remote.go

```diff
@@ -781,10 +781,23 @@ func (wd *remoteWD) SetAlertText(text string) error {
 }
 
 func (wd *remoteWD) execScriptRaw(script string, args []interface{}, suffix string) ([]byte, error) {
+	if args == nil {
+		args = make([]interface{}, 0)
+	} else {
+		for index, value := range args {
+			if ori, ok := value.(*remoteWE); ok {
+				args[index] = ori.toMap()
+			} else if ori, ok := value.(remoteWE); ok {
```

Comment:
Will the value ever be a non-pointer? In other words, will this branch ever execute?

I think we only ever emit the *remoteWE type as WebElements, so this branch can be removed.